### PR TITLE
#16: Simple webapp w/o login + #27: [WIP] Django Metrics View

### PR DESCRIPTION
**Pull request by shyammk**
Covers the work done for the Issue #16, Issue #27 and Issue #37 

- Simple Home, User Input and Metrics Display pages created
- Django Metrics view would need to be further modified when the code for calculating metrics is merged.
- The new output model was wired with the django framework

### CODE REVIEW

File: code/ui/lisibilite_ui/settings.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'hc)l7zu(fg*o_arf2(&&wj-)mkyr_j65+^#r5v*&b%6k*or0=1'
```
Comment (rupdeb) [inline code/ui/lisibilite_ui/settings.py:24]: in future we have to move this to a env.py sort of file

Comment (shyammk) [reply code/ui/lisibilite_ui/settings.py:24]: Agreed. Need to check how many such items need to be moved.

**Comment by rupdeb** [comment]
Looks fine, once the integration is done then we can fix if there are any issues

File: code/ui/lisibilite_ui_app/templates/scorecalculation.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
+{% load static %}
```
Comment (ats0stv) [inline code/ui/lisibilite_ui_app/templates/scorecalculation.html:1]: Suggestion: Would be good to name it something like `userinput` or `textinput` - `scorecalculation` seems a bit off IMO

Comment (ats0stv) [reply code/ui/lisibilite_ui_app/templates/scorecalculation.html:1]: Query: What does load static do?

File: code/ui/lisibilite_ui_app/tests.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (ats0stv) [inline code/ui/lisibilite_ui_app/tests.py:3]: 😼 😉

File: code/ui/lisibilite_ui_app/views.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+	def get(self, request):
+		"""
+		The function to handle the HTTP GET request for home page.
+		:param request: The django HTTP request object
+		:return The render object
+		"""
+		return render(request, 'index.html')
+
+	def post(self, request):
```
Comment (ats0stv) [inline code/ui/lisibilite_ui_app/views.py:16]: If post is rendering none, do we need it? Do you see a future case?

File: code/ui/lisibilite_ui_app/views.py [hunk 5]
```diff
@@ -0,0 +1,119 @@
+from django.shortcuts import render, redirect
```
Comment (ats0stv) [inline code/ui/lisibilite_ui_app/views.py:1]: May be for next PR. There are three views in this file. Would prefer splitting it

File: code/ui/lisibilite_ui_app/templates/scorecalculation.html (hunk 2 above)
Comment (shyammk) [reply code/ui/lisibilite_ui_app/templates/scorecalculation.html:1]: Will do the renaming as you suggested. It was userinput.html initially. I had changed it to the new name in the recent commit.

load static is is the general way of managing static files (like images, css, js etc) in django. It basically loads the path where these static files are stored. The path is defined in settings.py. You can read more here - https://docs.djangoproject.com/en/3.0/howto/static-files/.

File: code/ui/lisibilite_ui_app/views.py (hunk 4 above)
Comment (shyammk) [reply code/ui/lisibilite_ui_app/views.py:16]: Not really. Added it as a part of the code skeleton in the beginning. Will remove it.

File: code/ui/lisibilite_ui_app/views.py (hunk 5 above)
Comment (shyammk) [reply code/ui/lisibilite_ui_app/views.py:1]: Sure. Something to ponder.

File: code/ui/lisibilite_ui_app/views.py (hunk 4 above)
Comment (shyammk) [reply code/ui/lisibilite_ui_app/views.py]: I have pushed the changes related to this.

File: code/ui/lisibilite_ui_app/templates/scorecalculation.html (hunk 2 above)
Comment (shyammk) [reply code/ui/lisibilite_ui_app/templates/scorecalculation.html]: Pushed the changes related to the renaming of the file.

File: code/ui/lisibilite_ui_app/tests.py (hunk 3 above)
Comment (shyammk) [reply code/ui/lisibilite_ui_app/tests.py:3]: 😰 🏃‍♂️
Will add them shortly.

**Pull request merged**